pkg/cosign/tlog.go

```diff
@@ -75,6 +78,18 @@ func getLogID(pub crypto.PublicKey) (string, error) {
 	return hex.EncodeToString(digest[:]), nil
 }
 
+func intotoEntry(ctx context.Context, signature, pubKey []byte) (models.ProposedEntry, error) {
+	e, err := types.NewProposedEntry(ctx, intotoProposedEntryKey, "", types.ArtifactProperties{
```

Comment:
nit: add a comment that passing `""` as version uses the DefaultVersion

Reply:
resolved to use the default version defined in the package, thought that would make more sense